fastlane_core/lib/fastlane_core/ui/interface.rb

```diff
@@ -135,8 +140,9 @@ def crash!(exception)
     #   stack trace
     # Basically this should be used when you actively catch the error
     # and want to show a nice error message to the user
-    def user_error!(error_message)
-      raise FastlaneError.new, error_message.to_s
+    def user_error!(error_message, options = {})
+      options[:show_github_issues] = true unless options.key?(:show_github_issues)
```

Comment:
Keeping the parameters like: 

```ruby
(error_message, options = {})
```

I feel like I would more typically see this operation (combine options with a set of defaults) as:

```ruby
options = { show_github_issues: true }.merge(options)
```